package.json

```diff
@@ -19,12 +19,14 @@
   },
   "devDependencies": {
     "claire": "0.4.1",
+    "eslint": "^7.32.0",
     "mocha": "^9.2.2",
     "nyc": "^15.1.0",
     "uglify-js": "2.8.x"
   },
   "scripts": {
     "test": "make test",
+    "lint": "eslint \"lib/**/*.js\" channel_api.js callback_api.js",
```

Comment:
Does this catch the generated files? Maybe they should also be ignored (although if they pass, no harm being done currently).